styles.css

```diff
@@ -64,39 +63,37 @@ color: #0074CC;
 text-decoration: none;
 }
 
-
 i.fa-solid{
   margin-top:1em;
   padding:0 2em 0.7em 0;
   color: #0074CC;
-  font-size:1.3rem;
+  font-size:1.5rem;
   background-color:#F1F2F3;
 }
 
+/*column2
+forms and links*/ 
 
 .col2 {
-  width: 25vw;
-  height:100vh;
-  border:5px solid black;
+  width:25em;
+  height:66em;
+  margin:0;
   display:inline-block;
-
 }
 
 .button-container{
-  height:10vh;
-  width: 25vw;
-  margin-bottom: 1.3em;
+  height:10em;
+  width: 24em;
+  margin-bottom: 0.5em;
   background-color: #F1F2F3;
-  padding: 1.5em;
   position:relative;
-  left:-1.8vh;
-  top:-2.5vh;
+  
 }
 
 
 .facebook {
```

Comment:
I see there's a lot of code duplication between `.facebook`,  `.google` and `.github`.  Could you try to reduce the duplication by moving reused styles into it's own class (e.g. button-sign-up) and only keep button specific styles in each specific class?